### PR TITLE
Adding Sunsynk integration documentation

### PR DESCRIPTION
**Pull request by jamesridgway**
## Proposed change

Add Sunsynk integration documentation

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/85893
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/4038
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/sunsynk.markdown [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+ha_iot_class: "Cloud Polling"
+ha_config_flow: true
+ha_domain: sunsynk
+ha_platforms:
+  - sensor
+ha_codeowners:
+  - '@jamesridgway'
+---
+
+The Sunsunk integration gets data from the [PowerView Inteless APIs](https://pv.inteless.com/) to allow you to get details from your Sunsynk inverter and integrate these in your Home Assistant installation.
```
Comment (frenck) [inline source/_integrations/sunsynk.markdown]:
```suggestion
The Sunsynk integration gets data from the [PowerView Inteless APIs](https://pv.inteless.com/) to allow you to get details from your Sunsynk inverter and integrate these in your Home Assistant installation.
```

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

**Comment by davidflypei** [comment]
@jamesridgway  The parent link for core is incorrect, and its going to one of my old PRs.

**Comment by frenck** [comment]
Closing PR as the parent PR has been closed.

**Pull request closed without merging**